src/nunit.analyzers.tests/DiagnosticSuppressors/DereferencePossiblyNullReferenceSuppressorTests.cs

```diff
@@ -36,10 +43,10 @@ private class B
         private static readonly DiagnosticSuppressor suppressor = new DereferencePossiblyNullReferenceSuppressor();
 
         [TestCase("")]
-        [TestCase("Assert.NotNull(string.Empty)")]
-        [TestCase("Assert.IsNull(s)")]
-        [TestCase("Assert.Null(s)")]
-        [TestCase("Assert.That(s, Is.Null)")]
+        [TestCase("ClassicAssert.NotNull(string.Empty)")]
+        [TestCase("ClassicAssert.IsNull(s)")]
+        [TestCase("ClassicAssert.Null(s)")]
+        [TestCase("ClassicAssert.That(s, Is.Null)")]
```

Comment:
Should these 4 occurrences also use `$"{ClassicAssert}` instead of `ClassicAssert`

Reply:
Thanks for noticing.

The DiagnosticSuppressors work on the Syntax level (which is text) and don't know about the `using ClassicAssert = NUnit.Framework.Assert;` alias. Initially I changed some cases, but then decided to support both nunit versions in the analyzer:

```
private static bool IsValidatedNotNullByExpression(string possibleNullReference, ExpressionSyntax expression)
{
    // Check if this is an Assert for the same symbol
    if (AssertHelper.IsAssert(expression, out string member, out ArgumentListSyntax? argumentList) ||
        AssertHelper.IsClassicAssert(expression, out member, out argumentList))
    {
```

Al other code refers to this as `ClassicAssert`, so I will remove the constant.